opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/services/ServiceContextProvider.java

```diff
@@ -57,11 +65,41 @@
     ) {
         if (null != authSession) {
             return authSession.getParent();
-        } else {
-            ServiceSession session = new ServiceSession();
-            session.setContext(MAPPER.writeValueAsString(request));
-            return serviceSessions.save(session);
         }
+
+        FacadeServiceableRequest facadeServiceable = request.getFacadeServiceable();
+        String sessionPassword = facadeServiceable.getSessionPassword();
+
+        if (facadeServiceable.getXRequestID() == null) {
+            throw new RuntimeException("no request id");
+        }
+
+        Optional<ServiceSession> savedSession = serviceSessions.findById(facadeServiceable.getXRequestID());
+        if (savedSession.isPresent()) {
+            ServiceSession serviceSession = savedSession.get();
+            if (StringUtils.isEmpty(serviceSession.getPassword())) {
```

Comment:
Spring StringUtils are forbidden to use. We shall use Apache Commons.
And there is a better method `isBlank`

Reply:
ok